cmd/ipfs-cluster-ctl/main.go

```diff
@@ -860,6 +860,27 @@ but usually are:
 				},
 			},
 		},
+		{
+			Name:        "ipfs",
+			Usage:       "Manage IPFS daemon",
+			Description: "Manage IPFS daemon",
+			Subcommands: []cli.Command{
+				{
+					Name:  "gc",
+					Usage: "run garbage collection on ipfs repos of cluster peers",
+					Description: `
+This command would sweep the local set of stored objects and remove ones that
```

Comment:
`This command will instruct all current Cluster peers to run "repo gc" on their respective IPFS daemons`.